src/inputs/timePicker/timePicker.scss

```diff
@@ -20,7 +20,7 @@
         top: 64px;
         width: 260px;
         z-index: 1;
-        .ui.dropdown.dropdown-selection { margin-top: 0; }
+        .cmui.select { margin-top: 0; }
```

Comment:
Do we want to risk converting this whole SCSS file into JSS? we could stil use the classNames?

Reply:
I am willing to take a whack at it.  I did do just about everything else, eh? ;-)  I just thought that could wait to a later pass, and that I wasn't maybe the best qualified, but I'll see if I can bang it out.

Reply:
Well, I can checkout your branch and do it if we decide we want to do this. Up to you.

Reply:
... I pressed comment too soon.

I don't think want to block this PR just because of this issue. It's just a nice to have basically. Not really anything more than that.

Reply:
Thanks.  Appreciate the consideration.